database/migrations/2023_09_21_171540_adding_column_to_assigments_student.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up()
+    {
+        Schema::table('assignment_students', function (Blueprint $table) {
```

Comment:
Combine this two insertions into assignment_student: 
`public function up()
{
    Schema::table('assignment_students', function (Blueprint $table) {
        $table->string('assigment_file_path'); // Adding a string column
        $table->string('assigment_description'); // Adding another string column
    });
}
`